src/features/message-preview-content/MessagePreviewContent.js

```diff
@@ -1,14 +1,28 @@
 // // @flow
 import * as React from 'react';
 import classNames from 'classnames';
-import ContentPreview from '../../../../../elements/content-preview';
-import type { Token } from '../../../../../common/types/core';
-import type { ContentPreviewProps } from '../../../types';
-import Cache from '../../../../../utils/Cache';
-import PreviewGhost from './PreviewGhost';
+import ContentPreview from '../../elements/content-preview';
```

Comment:
My thought regarding simplifying the imports for props would look something like this, which would eliminate the direct reference to ContentPreview in all the consuming components:

```
// MessagePreviewContent.js
import ContentPreview, { type ContentPreviewProps } from '../../elements/content-preview';
export type ContentPreviewProps = ContentPreviewProps;

// PreviewTitleBodyTag.js (and others)
import MessagePreviewContent, { type ContentPreviewProps } from '../../../../message-preview-content/MessagePreviewContent';